--- a/libs/libwebsockets/Makefile
+++ b/libs/libwebsockets/Makefile
@@ -37,6 +37,10 @@
 CMAKE_OPTIONS += -DCMAKE_BUILD_TYPE=Release
 CMAKE_OPTIONS += -DLWS_WITH_EXTERNAL_POLL=ON
 CMAKE_OPTIONS += -DLWS_WITH_ULOOP=ON
+CMAKE_OPTIONS += -DLWS_WITHOUT_EXTENSIONS=OFF
+CMAKE_OPTIONS += -DLWS_WITH_ZLIB=ON
+CMAKE_OPTIONS += -DLWS_WITH_LIBEV=ON
+CMAKE_OPTIONS += -DLWS_WITH_EVLIB_PLUGINS=OFF
 
 # turn off all test apps
 CMAKE_OPTIONS += -DLWS_WITHOUT_TESTAPPS=ON
@@ -52,7 +56,7 @@
 	SECTION:=libs
 	CATEGORY:=Libraries
 	TITLE:=libwebsockets
-	DEPENDS:=+zlib +libcap +libubox
+	DEPENDS:=+zlib +libcap +libubox +libev
 	URL:=https://libwebsockets.org
 	MAINTAINER:=Karl Palsson <karlp@etactica.com>
 endef
